src/components/dialogs/affiliation-error.vue

```diff
@@ -1,19 +1,19 @@
 <template>
-  <v-dialog v-model="showModal" max-width="60%">
+  <v-dialog v-model="getAffiliationErrorModalVisible" max-width="60%" persistent>
```

Comment:
Persistent so that user must click OK to close this dialog. Otherwise, the visibility state is not reset and this modal does not re-display on subsequent errors.